examples/main.tf

```diff
@@ -0,0 +1,11 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "aws_ecr" {
+#  source = "git@github.com:ministryofjustice/cloud-platform-terraform-ecr-credentials.git"
```

Comment:
I'd actually use the git URL for `source` in the example here, as that's what it should be in real world use - I'd also add `?ref=master` at the end so that it's explicitly tracking the `master` branch, at least until we decide on how to handle versioning for our modules.

Reply:
You mean the `https` URL?

Reply:
Actually I just mean using a remote reference rather than a local path - but from looking at the docs I think the [Github version](https://www.terraform.io/docs/modules/sources.html#github) would be best?

Reply:
Ah I see, I was looking at the commented line. Yes, we should not be using a local path in the example since we expect people to just copy the blob of code.

As for the source URL, I agree, it should include a `ref=` (to `master` as you said, if we don't want to decide on versioning now) and I think we should use the `https` URL instead since not  every user will be able to read from a `git+ssh` URL.

(same for L9 in the README)

Reply:
@ale-novo ^